google-cloud-compat-checker/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Compatibility Checker
```

Comment:
I would probably want other utility stuff to be in a separate module - this module is likely to have weird dependencies (for example, a compile dependency on a maven plugin package).